app/src/androidTest/java/org/mozilla/focus/activity/SwitchContextTest.kt

```diff
@@ -45,159 +41,101 @@ class SwitchContextTest {
-    }
-
-    @Test
-    @Throws(UiObjectNotFoundException::class)
-    fun eraseAndOpenTest() {
-
+    fun notificationOpenButtonTest() {
         // Open a webpage
-        TestHelper.inlineAutocompleteEditText.waitForExists(waitingTime)
-        Assert.assertTrue(TestHelper.inlineAutocompleteEditText.exists())
-        TestHelper.inlineAutocompleteEditText.clearTextField()
-        TestHelper.inlineAutocompleteEditText.text =
-            webServer!!.url(TEST_PATH).toString()
-        TestHelper.hint.waitForExists(waitingTime)
-        pressEnterKey()
-
-        // Assert website is loaded
-        waitForWebSiteTitleLoad()
-
-        // Switch out of Focus, pull down system bar and select open action
+        searchScreen {
+        }.loadPage(webServer.url("").toString()) { }
+        // Send app to background
         pressHomeKey()
-        openNotification()
-
-        // If simulator has more recent message, the options need to be expanded
-        expandNotification()
-        TestHelper.notificationEraseOpenItem.click()
-
-        // Verify that it's on the main view, showing the initial view
-        TestHelper.erasedMsg.waitForExists(waitingTime)
-        Assert.assertTrue(TestHelper.erasedMsg.exists())
-        Assert.assertTrue(TestHelper.inlineAutocompleteEditText.exists())
-        Assert.assertTrue(TestHelper.initialView.exists())
+        // Pull down system bar and select Open
+        mDevice.openNotification()
+        notificationTray {
+            verifySystemNotificationExists(getStringResource(R.string.notification_erase_text))
+            expandEraseBrowsingNotification()
+        }.clickNotificationOpenButton {
+            verifyBrowserView()
+        }
     }
 
-    @Suppress("LongMethod")
     @Test
-    @Throws(UiObjectNotFoundException::class)
-    fun settingsToFocus() {
-
+    fun switchFromSettingsToFocusTest() {
         // Initialize UiDevice instance
         val LAUNCH_TIMEOUT = 5000
         val SETTINGS_APP = "com.android.settings"
-        val settings = TestHelper.mDevice.findObject(
+        val settingsApp = mDevice.findObject(
             UiSelector()
                 .packageName(SETTINGS_APP)
                 .enabled(true)
         )
+        val launcherPackage = mDevice.launcherPackageName
+        val context = InstrumentationRegistry.getInstrumentation().targetContext.applicationContext
+        val intent = context.packageManager
+            .getLaunchIntentForPackage(SETTINGS_APP)
 
         // Open a webpage
-        TestHelper.inlineAutocompleteEditText.waitForExists(waitingTime)
-        TestHelper.inlineAutocompleteEditText.clearTextField()
-        TestHelper.inlineAutocompleteEditText.text = webServer!!.url(TEST_PATH).toString()
-        TestHelper.hint.waitForExists(waitingTime)
-        pressEnterKey()
-
-        // Assert website is loaded
-        waitForWebSiteTitleLoad()
+        searchScreen {
+        }.loadPage(webServer.url("").toString()) { }
```

Comment:
What is this testing here?

Reply:
it's entering a URL in the search bar and loading the page. that's all